README.md

```diff
@@ -1,19 +1,20 @@
 # `atomvvm_packbeam`
 
-An Erlang Escript and OTP library used to generate an <a href="http://github.com/bettio/AtomVM">AtomVM</a> AVM file from a set of files (beam files, previously built AVM files, or even arbitrary data files).
+An Erlang Escript and library used to generate an [AtomVM](http://github.com/atomvm/AtomVM) AVM file from a set of files (beam files, previously built AVM files, or even arbitrary data files).
 
 This tool roughly approximates the functionality of the AtomVM `PackBEAM` utility, except:
 
 * Support for multiple data types, include beam files, text files, etc
-* "Smart" extraction of beams from AVM files, so that only the beams that are needed are packed
+* "Pruned" extraction of beams from AVM files, so that only the beams that are needed are packed
+* Support for embedded OTP applications in your PackBEAM files.
 
-The `packbeam` tool may be used on its own.  More typically, it is used internally as part of the <a href="https://github.com/fadushin/atomvm_rebar3_plugin">atomvm_rebar3_plugin</a> `rebar3` plugin.
+The `packbeam` tool may be used on its own as a stand-alone command-line utility.  More typically, it is used internally as part of the [`atomvm_rebar3_plugin`](https://github.com/atomvm/atomvm_rebar3_plugin) [`rebar3`](https://rebar3.org) plugin.
 
-# Prerequisites
+## Prerequisites
 
-Building Packbeam requires Erlang/OTP 22 or later, for compatibility with AtomVM, as well as a local installation of `rebar3`.  Optionally, any recent version of `make` may be used to simplify builds.
+Building `packbeam` requires Erlang/OTP 22 or later, for compatibility with AtomVM, as well as a local installation of [`rebar3`](https://rebar3.org).  Optionally, any recent version of `make` may be used to simplify builds.
```

Comment:
Do you want to change this to OTP24, since that is the oldest release you test in the CI?